frontend/src/components/messstelle/optionsmenue/panels/ZeitPanel.vue

```diff
@@ -173,32 +173,82 @@ const isAnwender = computed(() => {
 });
 
 const minDate = computed(() => {
-    return messstelleInfo.value.realisierungsdatum ?? "";
+    if (messstelleInfo.value.realisierungsdatum >= "2006-01-01")
+        return messstelleInfo.value.realisierungsdatum;
+    else return "2006-01-01";
 });
 
 const maxDate = computed(() => {
-    return messstelleInfo.value.abbaudatum ?? "";
+    const yesterday = new Date(new Date().setDate(new Date().getDate() - 1))
+        .toISOString()
+        .slice(0, 10);
+    return messstelleInfo.value.abbaudatum ?? yesterday;
 });
 
-const getFormattedSelectedZeit = computed(() => {
-    const zeitraum = chosenOptionsCopyZeitraum.value.slice(0);
-    if (zeitraum.length == 1) {
-        return dateUtils.formatDate(chosenOptionsCopyZeitraum.value[0]);
-    } else if (zeitraum.length == 2) {
-        const sortedDates = dateUtils.sortDatesDescAsStrings(zeitraum);
-        return `${dateUtils.formatDate(
-            sortedDates[1]
-        )} - ${dateUtils.formatDate(sortedDates[0])}`;
-    } else {
-        return "";
-    }
+const formattedDateInput = computed({
+    get: () => {
+        const zeitraum = chosenOptionsCopyZeitraum.value.slice(0);
+        if (zeitraum.length == 1) {
+            return dateUtils.formatDate(chosenOptionsCopyZeitraum.value[0]);
+        } else if (zeitraum.length == 2) {
+            const sortedDates = dateUtils.sortDatesDescAsStrings(zeitraum);
+            return `${dateUtils.formatDate(
+                sortedDates[1]
+            )} - ${dateUtils.formatDate(sortedDates[0])}`;
+        } else {
+            return "";
+        }
+    },
+    set: (newValue) => {
+        let newDate = newValue.replace(/\s/g, "");
+        if (newDate.length == 10) {
+            chosenOptionsCopy.value.zeitraum = [
+                dateUtils.formatDateToISO(newDate),
+            ];
+        } else if (newDate.length == 21) {
+            const [datum1, datum2] = newDate.split("-");
+            chosenOptionsCopy.value.zeitraum = [
+                dateUtils.formatDateToISO(datum1),
+                dateUtils.formatDateToISO(datum2),
+            ];
+        }
+    },
 });
 
 function allowedDatesRangeDatePicker(val: string) {
     const today = new Date();
     return new Date(val) < today;
 }
 
+function RULE_EINGABE_HAT_PLAUSIBLES_FORMAT() {
+    const dates = chosenOptionsCopyZeitraum.value;
+    const falschesFormat = "Datumsformat ist nicht korrekt";
+    const keinDatum = "Dieses Datum existiert nicht";
+    const dateRegex = /^\d{4}-\d{2}-\d{2}$/;
+    if (dates.length == 1) {
+        const dateObj = new Date(dates[0]).toString();
+        if (!dateRegex.test(dates[0])) {
+            return falschesFormat;
+        }
+        if (dateObj == "Invalid Date") {
+            return keinDatum;
+        }
+        return true;
+    }
+    if (dates.length == 2) {
+        let dateObj1 = new Date(dates[0]).toString();
+        let dateObj2 = new Date(dates[1]).toString();
```

Comment:
Bitte als const machen

Reply:
siehe commit [75bb1c7](https://github.com/it-at-m/dave-frontend/pull/131/commits/75bb1c70e597b7bf50bfcb92f4b308a61faa18eb)